### ISSUE TITLE
Migration to "opendata.cern"?

### ISSUE THREAD

**Issue by RaoOfPhysics** [cernopendata/opendata.cern.ch#826]
Title says it all. See https://nic.cern/registration-policy/

**Comment by tiborsimko** [comment]
Yes, we naturally considered that.  The last time I've heard, the process for the registration of subdomains wasn't really immediately available yet though. Should recheck...

**Comment by RaoOfPhysics** [comment]
@tiborsimko: It's now available -- https://cern.service-now.com/service-portal/function.do?name=Domain-Management

**Comment by psaiz** [comment]
This one seems to have gone under the radar for some time. Shall we look into this again? I'm happy taking over the ticket

**Comment by psaiz** [comment]
RQF2512641

**Comment by RaoOfPhysics** [comment]
Thanks for following up on this @psaiz! I’m no longer at CERN but super excited to see this implemented. <3

**Comment by psaiz** [comment]
It seems that they will discuss this request next week. I'll keep you posted

**Comment by psaiz** [comment]
And this is done! Closing the ticket.

**Comment by RaoOfPhysics** [comment]
NICE!!! :champagne:
